test/utils/helpers.js

```diff
@@ -1,3 +1,9 @@
 const isAscending = (arr, key) => arr.every((x, i) => x[key] <= arr[i][key])
 
-module.exports = { isAscending }
+const isInRange = (amount, range) => {
```

Comment:
if we can use this method in main service, it will simplify there also

Reply:
This is a good point but what we have in the main service is more performant. If we use this in the main service we will be splitting the same range for each result (and there could be thousands of results) where as in the main service we currently only do that once and reuse when we loop through results.